src/components/organisms/SettingsManager/Settings.tsx

```diff
@@ -75,6 +78,8 @@ export const Settings = ({
     Boolean(!config?.kubeConfig?.path) || Boolean(!config?.kubeConfig?.isPathValid)
   );
   const [currentKubeConfig, setCurrentKubeConfig] = useState(config?.kubeConfig?.path);
+  const [currentKubeConfigInValue, setCurrentKubeConfigInValue] = useState(config?.kubeConfig?.path);
```

Comment:
Why do we need to use this if we already have a `currentKubeConfig` ?

Reply:
I believe `currentKubeConfig` is used in some other place called `useDebounce` . So I added a new state to solve the confusion.
![image](https://user-images.githubusercontent.com/35147480/179514564-2e006761-8a4e-44fd-bcd8-19b6269810ea.png)

Reply:
Hmm, I think it doesn't change anything if it is used in `useDebounce`, we could still use it as the value for the input

Reply:
This `currentKubeConfigInValue`, Is used to set the `kubeconfig path` in `input` field after path is set to all place in the component.

So it matches with time delay.

So it is up to you whether you want this implementation or not